src/index.html

```diff
@@ -7,12 +7,58 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="cards"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="product-image"
+        class="cards_product-image"
+      />
+      <h1 class="cards_title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="cards_product_code">Product code: 195434</p>
+      <div class="cards_stars-content">
+        <section class="cards_stars">
+          <div class="cards_stars__star"></div>
+          <div class="cards_stars__star"></div>
+          <div class="cards_stars__star"></div>
+          <div class="cards_stars__star"></div>
+          <div class="cards_stars__star"></div>
```

Comment:
these are elements of `stars` block
```suggestion
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
```

Reply:
fixed